charts/datahub/subcharts/datahub-frontend/templates/ingress.yaml

```diff
@@ -29,6 +26,28 @@ spec:
   rules:
   {{- range .Values.ingress.hosts }}
     - host: {{ .host | quote }}
+  {{- if eq $ingressApiVersion "networking.k8s.io/v1" }}
```

Comment:
Seems like the change here is to either use backend.service instead of backend. Can we just put the if statement in those two places? Just to clearly specify the change between the if cases here.

Reply:
I can, but there is `networking.k8s.io/v1` for >=1.19 - new syntax
but for <1.19 there are 2 options (`extensions/v1beta1` and `networking.k8s.io/v1beta1`) - old syntax.
so new if statement will look like this:
```
{{- if or (eq $ingressApiVersion "extensions/v1beta1") (eq "networking.k8s.io/v1beta1") }}
```
instead of:
{{- else }}

What do you think about that?

Reply:
Oh. the if statement logic as is is fine. Just wanted to push the if check down to make it clear what is the diff between the if clause and the else clause

Reply:
I added that. Could you take a look and resolve?

Reply:
Sry I should've been more clear. The following was what I was thinking
```
      http:
        paths:
        {{- range .redirectPaths }}
          - path: {{ .path }}
            pathType: ImplementationSpecific
            backend:
              {{- if eq $ingressApiVersion "networking.k8s.io/v1" }}
              service:
                name: {{ .name }}
                port:
                  number: {{ .port }}
              {{- else }}
              serviceName: {{ .name }}
              servicePort: {{ .port }}
              {{- end }}
        {{- range .paths }}
          - path: {{ . }}
            pathType: ImplementationSpecific
            backend:
              {{- if eq $ingressApiVersion "networking.k8s.io/v1" }}
              service:
                name: {{ $fullName }}
                port:
                  number: {{ $svcPort }}
              {{- else }}
              serviceName: {{ $fullName }}
              servicePort: {{ $svcPort }}
              {{- end }}
        {{- end }}
```

Reply:
Sure. I've changed it. Could you take a look now?